internal/internal_task_handlers.go

```diff
@@ -391,6 +392,15 @@ OrderEvents:
 		default:
 			if isPreloadMarkerEvent(event) {
 				markers = append(markers, event)
+			} else if attrs := event.GetWorkflowExecutionUpdateAcceptedEventAttributes(); attrs != nil {
+				msgs = append(msgs, &protocolpb.Message{
```

Comment:
Would have preferred to have kept inferMessage for here (but adjusted to operate on a single event) but that can be a refactor.